webpack.config.umd.js

```diff
@@ -10,26 +8,30 @@ module.exports = {
     filename: "index.js",
     libraryTarget: "umd"
   },
+  resolve: {
+    extensions: [".js", ".jsx"]
+  },
   module: {
     rules: [
       {
         loader: "babel-loader",
         test: /\.(js|jsx)$/,
         exclude: /node_modules\/proptypes|scripts\/sw.js/,
         options: {
-          presets: [["es2015", { modules: false }]],
+          babelrc: false,
+          presets: ["es2015", "react", "stage-0"],
           plugins: [
-            ["transform-react-jsx", { pragma: "h" }],
-            "transform-object-rest-spread"
+            ["transform-react-jsx", { pragma: "h" }]
           ]
         }
       }
     ]
   },
   plugins: [
-    new CssMigrationWebpackPlugin(),
```

Comment:
It's still under one command, the `npm run build` still works like before. I'm modeling these build steps from the [react-virtualized](https://github.com/bvaughn/react-virtualized/blob/master/package.json) project